src/pages/Homepage/MonthlyGoodHomepage.vue

```diff
@@ -363,6 +370,7 @@ export default {
 			isSticky: false,
 			initialBottomPosition: 0,
 			mgStickBarOffset: 0,
+			showMonthlyGoodSelectorSection: false,
```

Comment:
Please put this in a Data object coming out of Contentful (`ui-homepage-monthly-good`. There is a JSON object there you can use which will be available to pass down from Homepage > MG Homepage as a prop.

This way the configuration won't be hard-coded and require a release.

Reply:
please make this configurable externally with the next phase.